### DISCUSSION TITLE
Deploy with a FHIR server on AKS

### DISCUSSION THREAD

**Discussion by Alan-Kuan** [microsoft/Graphir#63] [Q&A]
Hello everyone,

I deployed Microsoft's FHIR server on an AKS cluster. I found that the document of this repo only showed how to deploy Graphir with Azure API for FHIR. I wonder whether there's a way to deploy Graphir with a FHIR server on AKS.

**Comment by pjirsa** [comment]
Hi Alan. Graphir can be configured to point to any FHIR R4 compliant API. You just need to update the BaseUrl in the appsettings file.
```
 "FhirConnection": {
    "BaseUrl": "https://chestist-fhir-api.azurehealthcareapis.com", <--- Your FHIR server URL
    "Scopes": "https://chestist-fhir-api.azurehealthcareapis.com/user_impersonation"
  },
```

**Reply by Alan-Kuan** [reply]
Hi Phil,

Thanks for your help. However, I didn't figure out where was the `appsettings.json`. Should I clone the repo, and edit it in VS? Then, somehow deploy it on Azure?

I followed [the steps to register a resource application](https://github.com/microsoft/Graphir/blob/main/docs/authentication.md#register-a-resource-application), and I stucked at the step 9, where I had to update the value of `appsettings.json`.
By the way, what value should I fill when creating the `user_impersonation` app role?
